### ISSUE TITLE
Multiplying Money by Decimals - unusual behaviour

### ISSUE THREAD

**Issue by DavidPratten** [CatalaLang/catala#309]
The assertion in the Catala program below fails. 

The trace shows that the answer found was `CalculateGST.gst_amount: $90.91`

Any pointers on what I have missed will be appreciated.

David

### GST Article
The price charged for goods and services will include GST. The amount of GST included is 1/11 of the price charged. 

```catala
declaration structure GoodOrService:
  data price content money

declaration scope CalculateGST:
  input item content GoodOrService
  output gst_amount content money

scope CalculateGST:
  definition gst_amount equals
    item.price *$ 0.0909090909  # 1/11 represented as a decimal

declaration scope TestGST:
  gst scope CalculateGST
  output gst_amount content money

scope TestGST: # catala Interpret --scope=TestGST gst.catala_en
  definition
    gst.item
  equals
    GoodOrService {
      -- price: $100.00
    }
  definition gst_amount equals gst.gst_amount
  assertion gst_amount = $9.09
```

**Comment by DavidPratten** [comment]
BTW this appears to be a lexical issue.  

The following works perfectly.

```
scope CalculateGST:
  definition gst_amount equals
    item.price *$ 1.0 /. 11.0
```

**Comment by denismerigoux** [comment]
This issue is related to the old bug fixed by #249 and #256. The latest version of Catala does not exhibit it anymore.
